src/main/java/sirius/kernel/xml/NoContentNodeHandler.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.kernel.xml;
+
+/**
+ * Can be used to handle XML nodes without having the parser to read out the full content of the node, but only
+ * the node with attributes. See {@link NodeHandler#ignoreContent()} for an example.
+ */
+public abstract class NoContentNodeHandler implements NodeHandler {
```

Comment:
I would then rename to NoChildrenNodeHandler, so its clear that the Node contains all attributes and text value.

Reply:
Actually it does not, see latest commit for clarification inside javadoc example. In case we need a more sophisticated handling with xml mixed mode nodes, I suggest we add that at a later point.